core/src/main/java/com/linecorp/armeria/common/logging/LogSanitizers.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2022 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.common.logging;
+
+import java.util.function.BiFunction;
+
+import com.linecorp.armeria.common.HttpHeaders;
+import com.linecorp.armeria.common.RequestContext;
+import com.linecorp.armeria.common.annotation.Nullable;
+import com.linecorp.armeria.common.util.Functions;
+
+/**
+ * A holder class that has sanitizers for HTTP request/response header, content, and trailer.
+ */
+public class LogSanitizers<T extends HttpHeaders> {
```

Comment:
As in this class is responsible for sanitizing logs in general (instead of simply acting as a holder)

```suggestion
public class LogSanitizer<T extends HttpHeaders> {
```

Reply:
Also, does this class need to have a generic type parameter?